rotors_hil_interface/include/rotors_hil_interface/hil_interface_node.h

```diff
@@ -25,43 +25,51 @@ namespace rotors_hil {
 // Default values
 static constexpr bool kDefaultSensorLevelHil = true;
 static constexpr double kDefaultHilFrequency = 100.0;
-static constexpr double kDefaultBodyToSensorsRoll = M_PI;
```

Comment:
My only question is here: on MAVROS, we apply ENU to NED transform (https://github.com/mavlink/mavros/blob/master/mavros/src/lib/ftf_frame_conversions.cpp#L30) and then base_link to body/aircraft frame (https://github.com/mavlink/mavros/blob/master/mavros/src/lib/ftf_frame_conversions.cpp#L37), while you just seem to apply this last one. Which frame of reference are we using in `rotors_gazebo`?

Reply:
I believe Rotors uses the standard ROS reference frame of NWU, hence the rotation by 180 degrees around the x axis to convert it to NED in the HIL interface node in this case.